### DISCUSSION TITLE
Is there any way to add some photos in one message? (I'm a beginner developer.)

### DISCUSSION THREAD

**Discussion by relaxsus** [MessageKit/MessageKit#1698] [Q&A]
## General Information

* MessageKit Version:

* iOS Version(s):

* Swift Version:

* Devices/Simulators:

* Reproducible in ChatExample? (Yes/No):

**Comment by SD10** [comment]
@relaxsus Do you mean multiple photos in a single Message? We don’t support anything like that at the moment

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>
